source/introduction.txt

```diff
@@ -0,0 +1,243 @@
+providers. You can enable authentication providers in your server-side configuration then immediately log in from any client application using a {+backend-short+} SDK.
+
+
+To learn more about authentication in {+backend+}, see :ref:`users-and-authentication`.
+
+.. _intro-web-schema:
+
+Schema Validation and Data Access Rules
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Modern applications require that data is available using consistent types and formats.
+Data consistency guarantees that all the application's components can work together.
+It's also important to make sure that any given piece of data is only accessible
+to an authorized user.  For example, you might grant a user access
+to only their own data.
+
+{+backend+} validates data with a {+backend-short+} schema you define in the `JSON schema <https://json-schema.org/>`__ standard.
+A {+backend-short+} schema serves as the source of truth for defining data types in your application.
+{+backend-short+} also uses the schema to map data between your application and a 
+`{+atlas+} <https://www.mongodb.com/realm?tck=docs_realm#atlas-form-container>`_ collection.
+
+You can also configure {+backend-short+} to secure data with role-based data access rules.
+These rules determine each user's access to every document for every request.
+A user can only insert, read, or modify data if both they and the document
+meet pre-defined conditions.
+
+
+To learn more about defining a schema and access rules for your data, see
+:ref:`collection rules <mongodb-rules>`.
+
+.. _intro-web-functions:
+
+Event-Driven Serverless Functions
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Many applications require that some business logic runs on a server. Usually this
+logic deals with sensitive user data or interacting with external services. Apps might
+also need to respond in real time when something happens in the app or in an external service.
+
+{+backend+} lets you define serverless JavaScript functions called {+backend-short+} Functions.
+You can use Realm Functions for the following common backend use cases:
+
+- Call from client applications with the Realm Web SDK.
+- Execute Realm Functions in response to trigger events, like an update to a document in MongoDB or at a scheduled time.
+- HTTPS endpoints that execute a function in response to external applications.
+
+To learn more about defining and using serverless functions, see
+:ref:`functions` and :ref:`triggers`.
+
+.. _intro-web-query:
+
+Secure Client-Side Queries
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Developers need efficient, secure data access.
+If the data is on a server, you need an API to access the data and prevent unauthorized access.
+
+The Realm Web SDK includes a :ref:`secure MongoDB client <intro-web-mql>` to interact with data. It uses
+:ref:`GraphQL <intro-web-graphql>` and the :manual:`Query API </tutorial/query-documents>`
+to access data in {+atlas+}. Use either or both of these APIs to work with
+your application's data.
```

Comment:
we could generalize it a bit more to be less web-centric, more general client-side talking about other SDKs